src/EnumDefinition.php

```diff
@@ -0,0 +1,31 @@
+<?php
+
+namespace Spatie\Enum;
+
+class EnumDefinition
+{
+    /** @var string|int */
+    public $value;
+
+    public string $label;
+
+    private string $methodName;
+
+    /**
+     * @param string $methodName
+     * @param string|int $value
+     * @param string $label
+     */
+    public function __construct(string $methodName, $value, string $label)
+    {
+        $this->methodName = strtolower($methodName);
+        $this->value = $value;
+        $this->label = $label;
+    }
+
+    public function equals($input): bool
```

Comment:
I would like to reference your guidelines and prefer single short `if`s over this line-breaked `or` one.

Reply:
Absolutely right, I've changed it!

Reply:
https://github.com/spatie/enum/pull/56/commits/991120832d9b6a11b4170d2cf158a1873792f63b